tools/shared/src/main/scala/org/scalajs/core/tools/sem/Semantics.scala

```diff
@@ -247,7 +247,7 @@ object Semantics {
       productionMode = false,
       runtimeClassNameMapper = RuntimeClassNameMapper.keepAll())
 
-  def compliantTo(semantics: Traversable[String]): Semantics = {
+  def compliantTo(semantics: Iterable[String]): Semantics = {
```

Comment:
That is not an acceptable change, because it breaks a public API.

You need to use a `type` alias in an `Compat` object instead.